Changes

```diff
@@ -359,6 +359,9 @@ Working version
 - #9889: more caching when printing types with -short-path.
   (Florian Angeletti, review by Gabriel Scherer)
 
+-  #9599: fix pprint of extension constructors (and exceptions) that rebind
+   (Chet Murthy, review by octachron@)
```

Comment:
`Florian Angeletti` rather than `octachron@`. Also the Change entry is conflicting with the one in the previous PR. Do you want me to (force) push a fixed and rebased version?

Reply:
There's no point in you doing this sort of minor work; let me do it and I'll push an updated rebased branch.

Reply:
OK, pushed a rebased branch, builds, passes all tests.